napari/_tests/test_notebook_display.py

```diff
@@ -37,18 +37,18 @@ def test_nbscreenshot(make_napari_viewer):
         ("Good alt text", "Good alt text"),
         # Naughty strings https://github.com/minimaxir/big-list-of-naughty-strings
         # ASCII punctuation
-        (r",./;'[]\-=", ',./;&#x27;[]\\-='),  # noqa: W605
+        (r",./;'[]\-=", ',./;&#x27;[]\\-='),
         # ASCII punctuation 2, skipping < because that is interpreted as the start
         # of an HTML element.
         ('>?:"{}|_+', '&gt;?:&quot;{}|_+'),
         ("!@#$%^&*()`~", '!@#$%^&amp;*()`~'),  # ASCII punctuation 3
         # # Emojis
         ("😍", "😍"),  # emoji 1
         ("👨‍🦰 👨🏿‍🦰 👨‍🦱 👨🏿‍🦱 🦹🏿‍♂️", "👨‍🦰 👨🏿‍🦰 👨‍🦱 👨🏿‍🦱 🦹🏿‍♂️"),  # emoji 2
-        (r"¯\_(ツ)_/¯", '¯\\_(ツ)_/¯'),  # Japanese emoticon  # noqa: W605
+        (r"¯\_(ツ)_/¯", '¯\\_(ツ)_/¯'),  # Japanese emoticon
         # # Special characters
         ("田中さんにあげて下さい", "田中さんにあげて下さい"),  # two-byte characters
-        ("表ポあA鷗ŒéＢ逍Üßªąñ丂㐀𠀀", "表ポあA鷗ŒéＢ逍Üßªąñ丂㐀𠀀"),  # special unicode chars
+        ("表ポあA鷗ŒéB逍Üßªąñ丂㐀𠀀", "表ポあA鷗ŒéB逍Üßªąñ丂㐀𠀀"),  # special unicode chars
```

Comment:
What changed here? Are we ok with it or is it masking our test?

Reply:
Yes. It removed character an changed in normal `B`

Reply:
But the point of this test is to test those special characters, so maybe we should undo this?

Reply:
It was undone.